_includes/post-macros.html

```diff
@@ -675,118 +508,39 @@ <h1 class="h4">{{ icon }} {{ header }}</h1>
                 "From" date input
                 =================
             #}
-
-            {# No-JS date input #}
             {% set from_date = selected_filters_for_field('range_date_gte')[0] %}
-            <div class="form-l_col form-l_col-1-2 js-filter_range-date-wrapper">
+            {% set from_date_formatted = from_date | date('%m/%d/%Y') if from_date %}
+            <div class="form-l_col form-l_col-1-2">
                 <label for="{{ options.id_range_date_gte }}">
-                    From:
+                    From <span class="date_label">(YYYY-MM)</span>:
                 </label>
                 <input class="js-filter_range-date js-filter_range-date__gte"
                        id="{{ options.id_range_date_gte }}"
                        name="filter_range_date_gte"
                        type="text"
-                       value="{{ from_date if from_date }}"
+                       value="{{ from_date_formatted if from_date_formatted }}"
```

Comment:
What's the value of `from_date_formatted` if it's not present? If it's an empty string this could be shortened to `value="{{ from_date_formatted }}"`